exercises/java/src/main/java/com/adaptionsoft/games/trivia/runner/GameRunner.java

```diff
@@ -12,17 +12,19 @@ public class GameRunner {
 	public static void main(String[] args) {
 		Game aGame = new Game();
 		
-		aGame.add("Chet");
-		aGame.add("Pat");
-		aGame.add("Sue");
+		aGame.addPlayer("Chet");
+		aGame.addPlayer("Pat");
+		aGame.addPlayer("Sue");
 		
 		Random rand = new Random(Integer.parseInt(args[0]));
 	
 		do {
 			
 			aGame.roll(rand.nextInt(5) + 1);
+
+			boolean checkIfIncorrectAnswer = rand.nextInt(9) == 7;
 			
-			if (rand.nextInt(9) == 7) {
+			if (checkIfIncorrectAnswer) {

```

Comment:
Good use of variable to represent the wrong answer

Reply:
Thanks :)